packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -953,6 +955,20 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		return columnFilterValue;
 	}
 
+	/**
+	 * Get the position of the filter box of the specified column.
+	 * @param columnName - Name of the column whose filter box position should be retrieved.
+	 * @returns The position of the filter box of the specified column.
+	 */
+	public getColumnFilterPosition(columnName: string): StarkTableColumnFilter["filterPosition"] {
+		if (!(this.filter.columns instanceof Array)) {
+			return undefined;
+		}
+
+		const column = this.filter.columns.find((columnFilter: StarkTableColumnFilter) => columnFilter.columnName === columnName);
+		return column && column.filterPosition;
```

Comment:
This line still needs to fixed ;)

`return column ? column.filterPosition : undefined;`